### ISSUE TITLE
Thoughts on padded-blocks rule?

### ISSUE THREAD

**Issue by mattdesl** [feross/standard#86]
What are your thoughts on the padded-blocks rule?
http://eslint.org/docs/rules/padded-blocks.html

I have some code like this:

```js
function foo () {
  return doAsync()
    .then(function () {
      return ...
    }, function () {
      // user exited early 
      return ...
    })
}
```

But because of this rule I am getting an error because `// user exited early` is treated as a blank line.

Personally I find super strict rules like this just slow down my dev cycle.

**Comment by maxogden** [comment]
hmm wouldn't a fix be to treat comments as non-blank lines?

**Comment by Flet** [comment]
I was able to verify that this passes with no linting problems:

```js
function doAsync () {}

function foo () {
  return doAsync()
    .then(function () {
      return 'yay'
    }, function () {
      // user exited early
      return 'wee'
    })
}

module.exports = foo
```

Is it possible to run `standard --verbose` to see what rule is being violated for your snippet?

**Comment by mattdesl** [comment]
```
Error: Use JavaScript Standard Style (https://github.com/feross/standard)
  foobar.js:17:24: Block must not be padded by blank lines. (eslint/padded-blocks)
```

Exact code:

```js
var argv = {}
commit()

//commits current working dir, resolves to html_url
function commit (result) {
  var url = result.html_url
  //user opted not to commit anything
  if (argv.b || argv.bare) {
    return Promise.resolve(url)
  }
  return getMessage().then(function (message) {
    return gitCommit({
      message: message,
      url: url + '.git'
    }).catch(function () {
      return Promise.resolve(url)
    }).then(function () {
      // some stuff
      return url
    })
  })
}

function getMessage () {

}

function gitCommit () {

}
```

**Comment by maxogden** [comment]
For that snippet w/ latest npm `standard` I get:

```
$ standard
Error: Use JavaScript Standard Style (https://github.com/feross/standard)
  index.js:4:0: Expected space or tab after // in comment.
  index.js:7:2: Expected space or tab after // in comment.
  index.js:30:1: Newline required at end of file but not found.
```

**Comment by Flet** [comment]
@maxogden same! After fixing them it is clean.

**Comment by mattdesl** [comment]
I'm using latest as well (3.2.1). I get the newline issue but not the `space or tab after //`. 

I think this is related to the babel-eslint issue.

**Comment by mattdesl** [comment]
Resolved after #85 was fixed
